### PR TITLE
add option to MembershipType

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/10250

### CODE REVIEW

File: teams/teams-ps/teams/New-TeamChannel.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Accept wildcard characters: False
 ```
 
 ### -MembershipType
-Channel membership type, Standard or Private.
+Channel membership type, Standard, Shared or Private.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-TeamChannel.md:91]:
```suggestion
Channel membership type, Standard, Shared, or Private.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@dariomws Please review. Thanks

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**